packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -1383,6 +1383,8 @@ export interface OwidGdocDataInsightContent {
     type: OwidGdocType.DataInsight
 }
 
+export const DATA_INSIGHTS_INDEX_PAGE_SIZE = 2
```

Comment:
This should be 20 but I've made it artificially low while we're testing pagination